### DISCUSSION TITLE
Running mitmproxy on google shell

### DISCUSSION THREAD

**Discussion by Le-git-hub** [mitmproxy/mitmproxy#5824] [General]
hello, im trying to find a way to host mitmproxy online and not locally, i tried to use the command `mitmproxy --set block_global=false` and it worked on my personal pc with the public ipv6 on my local machine, but i couldn't find the ipv6 of the google shell, so i couldn't connect to the mitmproxy.
is there any way to see what is the proxy (instead of it showing *:8080)? 
Thanks

**Comment by rosydawn6** [comment]
Do you mean find the public ip address of the remote server?  If so there are a variety of ways to obtain that information: https://stackoverflow.com/questions/13322485/how-to-get-the-primary-ip-address-of-the-local-machine-on-linux-and-os-x

For a web api based approach I generally use:
`curl ifconfig.me`
